_topic_maps/_topic_map.yml

```diff
@@ -2692,8 +2692,10 @@ Topics:
     File: network-observability-operator-monitoring
   - Name: Scheduling resources
     File: network-observability-scheduling-resources
-  - Name: API reference
+  - Name: FlowCollector API reference
     File: flowcollector-api
+  - Name: FlowMetric API reference
+    File: flowmetric-api
   - Name: JSON flows format reference
```

Comment:
can we take the opportunity to rename `JSON flows format reference` to `Flows format reference` ? (or is this planned in another PR?)

Reply:
I'm renaming FlowCollector in this PR because I now need to differentiate between the two Operator APIs whereas before I didn't need to do that. I could honestly do this in my API doc PR too, but it only occured to me to do it in this PR because I'm adding the 2nd reference. So yea, I changed the Flow format title in the Flow format PR :)